pom.xml

```diff
@@ -114,6 +114,7 @@
     <module>versioned</module>
     <module>servers</module>
     <module>tools</module>
+    <module>clients/iceberg-views</module>
```

Comment:
This change should be in `clients/pom.xml`

Reply:
done